langserver/symbol.go

```diff
@@ -27,31 +27,37 @@ import (
 type query struct {
 	kind   lsp.SymbolKind
 	filter filterType
+	file   string
 	tokens []string
 }
 
 // parseQuery parses a user's raw query string and returns a
-// structured representation of the query.
-func parseQuery(q string) query {
-	var qu query
-	toks := tokenizer.Split(strings.ToLower(q), -1)
-	index := 0
-	for _, tok := range toks {
-		if tok == "" {
+// structured representation of the query. Possible queries are:
+func parseQuery(q string) (qu query) {
```

Comment:
Minor style issue, I would advise against changing the signature to return `(qu query)` over `query`. Just a do a `var qu query` inside.

See https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters.

However, this is an unexported func and you won't ever see it in godoc, so perhaps it's fine to keep as is. Up to you. I'm okay with it as is because it's unexported.